packages/graphql/src/typeDefs/Growth.js

```diff
@@ -160,12 +162,14 @@ module.exports = `
   }
 
   type SocialContent {
-    id: String!,
-    titleKey: String!,
-    detailsKey: String!,
+    id: String!
+    titleKey: String!
+    detailsKey: String!
+    title: String
```

Comment:
Affaik, we haven't discuss the need with community team to have untranslated title and key for social content sharing. Are you adding those for anticipated future needs ?

Reply:
Removed for now.